modules/cnv-creating-new-vm-from-cloned-pvc-using-datavolumetemplate.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-cloning-vm-using-datavolumetemplate.adoc
+
+[id="cnv-creating-new-vm-from-cloned-pvc-using-datavolumetemplate_{context}"]
+= Creating a new virtual machine from a cloned PersistentVolumeClaim by using a DataVolumeTemplate
+
+You can create a virtual machine that clones the PersistentVolumeClaim (PVC) of
+an existing virtual machine into a DataVolume. By referencing a
+`dataVolumeTemplate` in the virtual machine `spec`, the `source` PVC is cloned to
+a DataVolume, which is then automatically used for the creation of the virtual
+machine.
+
+[NOTE]
+====
+When a DataVolume is created as part of the DataVolumeTemplate of a virtual
+machine, the lifecycle of the DataVolume is then dependent on the virtual
+machine: If the virtual machine is deleted, the DataVolume and associated
+PVC will also be deleted.
+====
+
+.Prerequisites
+* A PVC of an existing virtual machine disk. The associated virtual machine must
+be powered down, or the clone process will be queued until the PVC is available.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+
+.Procedure
+
+. Examine the virtual machine you want to clone to identify the name and
+namespace of the associated PVC.
+
+. Create a YAML file for a `VirtualMachine` object. The following virtual
+machine example clones `my-favorite-vm-disk`, which is located in the
+`source-namespace` namespace. The `2Gi` DataVolume called `favorite-clone`
+is created from `my-favorite-vm-disk`.
++
+For example:
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachine
+metadata:
+  labels:
+    kubevirt.io/vm: vm-dv-clone
+  name: vm-dv-clone <1>
+spec:
+  running: false
+  template:
+    metadata:
+      labels:
+        kubevirt.io/vm: vm-dv-clone
+    spec:
+      domain:
+        devices:
+          disks:
+          - disk:
+              bus: virtio
+            name: root-disk
+        resources:
+          requests:
+            memory: 64M
+      volumes:
+      - dataVolume:
+          name: favorite-clone
+        name: root-disk
+  dataVolumeTemplates:
+  - metadata:
+      name: favorite-clone
+    spec:
+      pvc:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 2Gi
+      source:
+        pvc:
+          namespace: "source-namespace"
+          name: "my-favorite-vm-disk"
+----
+<1> The virtual machine to create.
+
+. Create the virtual machine with the PVC-cloned DataVolume:
++
+----
+$ oc create -f <vm-clone-datavolumetemplate>.yaml
+----
+
```

Comment:
Is there a next step for this procedure?

Reply:
I don't think so. The main part of that procedure is just editing the config file.